interpreter/cling/include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -286,21 +286,24 @@ namespace cling {
   template <class T>
   inline std::string printValue(std::unique_ptr<T> *val)
   {
-     return "std::unique_ptr -> " + printValue(val->get());
```

Comment:
Why don’t you check if val is a nullptr and then print?

Reply:
I could if that is desired but the output would then be doubly inconsistent.  In case of raw pointer the value of the pointer is always printed.  And then the printing of the case where the pointer is null would be different from the case is it non null (and then the rare case of a pointer to pointer which is nullptr be ambiguous :) ).